### PR TITLE
Update mktg content mms-96

### PR DESCRIPTION
**Pull request by TripleCISU**
Please review our [contribution guidelines](https://github.com/SonarSource/sonar-scanner-vsts/blob/master/contributing.md).

And please ensure your pull request adheres to the following guidelines: 

- [ ] Please explain your motives to contribute this change: what problem you are trying to fix, what improvement you are trying to make
- [ ] Use the following formatting style: [SonarSource/sonar-developer-toolset](https://github.com/SonarSource/sonar-developer-toolset#code-style)
- [ ] Provide a unit test for any code you changed
- [ ] If there is a [JIRA](https://jira.sonarsource.com/browse/VSTS) ticket available, please make your commits and pull request start with the ticket ID (VSTS-XXXX)

### CODE REVIEW

File: extensions/sonarqube/overview.md [hunk 1]
```diff
@@ -1,50 +1,59 @@
...
+This extension allows automatic analysis of all branches and pull-requests which enables early discovery of bugs and security vulnerabilities prior to a merge. Branches and PRs get their own Quality Gate status and results are pushed right to your project space. 
 
 ![Branches](img/branches.png)
 
-If you configure your build definition as a build validation for pull requests of that project (this can be done on "Branch policies"), SonarQube will also analyze the code changes and decorate the pull request with comments and overall status so that you can merge with confidence:
+It's easy to add non-disruptive code quality checks right into your Azure DevOps workflow. Simply add SonarQube to your build pipeline definition and you're on your way to only promoting quality code. SonarQube analyzes the code changes and decorates pull requests with comments and overall status -> _merge with confidence!_
+
+![Build Config](img/Build-pipeline.png)
+
+When a build is run on a branch, the extension automatically configures the analysis to be pushed to the relevant project branch on SonarQube.
```
Comment (christophelevis) [inline extensions/sonarqube/overview.md]: The picture below corresponds to a pull request.

> When a build is run on a branch or a Pull Request, the extension automatically configures the analysis to be pushed to SonarQube and to decorate the Pull Request with issues and Quality Gate status.

?

File: extensions/sonarqube/overview.md [hunk 2]
```diff
@@ -1,50 +1,59 @@
...
+### Automatically Analyze Branches and Decorate Pull Requests (SonarQube Commercial Editions)
+This extension allows automatic analysis of all branches and pull-requests which enables early discovery of bugs and security vulnerabilities prior to a merge. Branches and PRs get their own Quality Gate status and results are pushed right to your project space. 
 
 ![Branches](img/branches.png)
 
-If you configure your build definition as a build validation for pull requests of that project (this can be done on "Branch policies"), SonarQube will also analyze the code changes and decorate the pull request with comments and overall status so that you can merge with confidence:
+It's easy to add non-disruptive code quality checks right into your Azure DevOps workflow. Simply add SonarQube to your build pipeline definition and you're on your way to only promoting quality code. SonarQube analyzes the code changes and decorates pull requests with comments and overall status -> _merge with confidence!_
```
Comment (christophelevis) [inline extensions/sonarqube/overview.md]: Pull Requests with capital letters?

File: extensions/sonarqube/overview.md [hunk 3]
```diff
@@ -1,50 +1,59 @@
...
+
+![Build Config](img/Build-pipeline.png)
+
+When a build is run on a branch, the extension automatically configures the analysis to be pushed to the relevant project branch on SonarQube.
 
 ![PR-Decoration](img/pull-request-decoration.png)
 
 **Important note**: to activate pull request decoration, you must specify a user token in the "General Settings > Pull Requests" administration page of your project in SonarQube.
```
Comment (christophelevis) [inline extensions/sonarqube/overview.md]: Pull Request with capital letters?

File: extensions/sonarqube/overview.md [hunk 4]
```diff
@@ -1,50 +1,59 @@
...
 
-Example of a passing Quality Gate:
-![Passed Qualiy Gate](img/sq-analysis-report-passed.png)
+### Easy setup for Maven and Gradle projects
+For Java, analyzing your source code is also very easy. It only requires adding the **Prepare Analysis Configuration** task and checking the **Run SonarQube Analysis** option in the 'Code Analysis' panel in your Maven or Gradle task.
 
-Example of a failing Quality Gate:
-![Failed Qualiy Gate](img/sq-analysis-report-failed.png)
+## SonarQube Azure DevOps Marketplace Extension Details
+This extension provides the following features:
```
Comment (christophelevis) [inline extensions/sonarqube/overview.md]: Talking about "features" for the endpoint and the tasks looks weird to me.
Also, I'm not sure we should repeat Branch and Pull Request analysis among those bullet points.
Or maybe:

> This extension provides Branch and **Pull Request** analysis with:...

?

File: extensions/sonarqube/overview.md [hunk 5]
```diff
@@ -1,50 +1,59 @@
...
+* A dedicated **SonarQube EndPoint** that defines the SonarQube server to be used.
+* Branch and **Pull Request** analysis in your projects.
+* Three build tasks to get your projects analyzed easily:
+  * **Prepare Analysis Configuration:** Configures all the required settings prior to executing a build. This task is mandatory. For .NET solutions or Java projects, this task helps SonarQube seamlessly integrate with MSBuild, Maven and Gradle tasks.
+  * **Run Code Analysis:** Executes the source code analysis. This task isn't required for Maven or Gradle projects.
+  * **Publish Quality Gate Result:** Displays the Quality Gate status in the build summary. This tasks is optional, as it may increase the overall build time.
 
+**Note for TFS installations older than TFS 2017 Update 2**: to install the extension, please follow instructions
+available on the ["SonarQube Extension 3.0" documentation page](https://docs.sonarqube.org/display/SCAN/SonarQube+Extension+3.0).
```
Comment (christophelevis) [inline extensions/sonarqube/overview.md]: There's a line return between instructions and available.
Also, I can't find anymore the documentation page for v3. This link fallbacks on same page as the [getstarted] link provided below.  I guess that this was done on purpose and we can drop the note.

File: extensions/sonarqube/vss-extension.json [hunk 6]
```diff
@@ -24,7 +25,7 @@
   "icons": {
     "default": "extension-icon.png"
   },
-  "tags": ["build", "ci", "continuous integration", "code quality", "static code analysis"],
+  "tags": ["build", "ci", "continuous integration", "code quality", "static code analysis", "code analysis", "code quality", "SAST", "security"],
```
[the remark this answers is not in the document]
Comment (mickael-caro-sonarsource) [reply extensions/sonarqube/vss-extension.json]: I checked and we are not apparently. "static code analysis", "code analysis" are quite redundant each other, we can let only one.

File: extensions/sonarqube/vss-extension.json [hunk 7]
```diff
@@ -8,6 +8,7 @@
     "theme": "dark"
   },
   "publisher": "sonarsource",
+  "homepage": "https://www.sonarqube.org/"
```
Comment (mickael-caro-sonarsource) [inline extensions/sonarqube/vss-extension.json]: Missing coma at the end.

File: extensions/sonarqube/overview.md [hunk 8]
```diff
@@ -1,50 +1,59 @@
...
+
+![Passed Qualiy Gate](img/sq-analysis-report-passed.png)
+
+Example of a failing Quality Gate:
+
+![Failed Qualiy Gate](img/sq-analysis-report-failed.png)
 
-### Branch and Pull Request analysis
-Whatever type of source repository you are analysing, when a build is run on a branch of your project, the extension automatically configures the analysis to be pushed to the relevant project branch on SonarQube:
+### Automatically Analyze Branches and Decorate Pull Requests (SonarQube Commercial Editions)
```
[the remark this answers is not in the document]
Comment (TripleCISU) [reply extensions/sonarqube/overview.md]: Yup, I added some content that covers CE functionality and the additional functionality you get from SQ Commercial Editions.

**Comment by mickael-caro-sonarsource** [comment]
@TripleCISU removing the whole "screenshot" did the trick, no error and extension page look ok :)

**Pull request merged**